### ISSUE TITLE
feature request

### ISSUE THREAD

**Issue by drwetter** [drwetter/testssl.sh#1524]
https://community.letsencrypt.org/t/revoking-certain-certificates-on-march-4/114864

If anybody wants to implement a check for the affected serials (--phone-out should be mandatory), go ahead.

This issue expires probably soon.

**Comment by dcooper16** [comment]
If I understand correctly, this issue may already be overcome by events. According to the announcement, all of the affected certificates should have been revoked by now. It seems that Let's Encrypt certificates include OCSP URIs. So, if someone runs testssl.sh with `--phone-out`, testssl.sh should report that the certificate is revoked. It won't report the reason that the certificate is revoked, but does that matter?

**Comment by drwetter** [comment]
yes, sure, when they have been revoked, no action is needed. By the time I wrote this issue that was not the case. Then LE said, they will be revoked. Then I read, not every certificate will be revoked.

I haven't followed the last ~two days. What I found (https://bugzilla.mozilla.org/show_bug.cgi?id=1619179#c4) was that they are partly revoked and partly not. Don't know whether this is the latest info?

If not, we can close this. Also if otherwise this is true, this issue will naturally resolve itself soon I believe as the lifetime of those certificates is fairly limited.

If somebody still wants to implement this, the check is easy. https://letsencrypt.org/caaproblem/ . It'd take still some days until I would get to it.
